<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher</class>
 <widget class="QMainWindow" name="Teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AcadEref</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CEESS-教师端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="btnTestManage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>题库管理</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnUserManage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账号管理</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnScoreAnalyse">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>成绩分析</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnSetting">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>系统设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
